pkg/oci/errors.go

```diff
@@ -0,0 +1,20 @@
+package oci
+
+import "fmt"
+
+// BuildErr indicates a general build error occurred.
+type BuildErr struct {
+	Err error
+}
+
+func (e BuildErr) Error() string {
+	return fmt.Sprintf("error performing host build. %v", e.Err)
+}
+
+type ErrBuildInProgress struct {
+	Dir string
+}
+
+func (e ErrBuildInProgress) Error() string {
+	return fmt.Sprintf("Build directory already exists for this version hash and is associated with an active PID.  Is a build already in progress? %v", e.Dir)
```

Comment:
Nit: is there a way to make this consistent with other error messages? I.e. starting with a lowercase letter and no real punctuation? A suggestion - perhaps it could be better.
```suggestion
	return fmt.Sprintf("a build for this function is associated with an active PID appears to be already in progress %v", e.Dir)
```